app/controllers/v0/health_care_application_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+module V0
+  class HealthCareApplicationController < ApplicationController
+    skip_before_action(:authenticate)
+
+    def create
+      health_care_application = params[:form]
+
+      if health_care_application
+        render(json: { success: true })
+      else
+        render(json: { success: false })
```

Comment:
It looks like the JSON here is just for testing purposes, but I wanted to make mention that if a resource is created that does not have a real payload, then the HTTP status code 204 "No Content" with no JSON body is ideal.

for anything that is not a success, we would raise an error that application_controller.rb uses to render JSON according to how JSONAPI v1.0 spec suggests to do so.

This is just a comment for posterity sake. I realize this is just for testing purposes.